jdaviz/configs/default/plugins/model_fitting/model_fitting.vue

```diff
@@ -192,7 +192,29 @@
         action_tooltip="Fit the model to the data"
         :action_disabled="model_equation_invalid_msg.length > 0"
         @click:action="apply"
-      ></plugin-add-results>
+      >
+        <div v-if="config!=='cubeviz' || !cube_fit">
+          <v-row>
+            <v-switch
+              v-model="residuals_expose"
+              label="Expose residuals"
+              hint="Whether to compute and export residuals (data-model)."
```

Comment:
```suggestion
              hint="Whether to compute and export residuals (data - model)."
```
Is this perhaps more readable as "data minus model"?

Reply:
I like that even more (to avoid confusion with hyphenation), thanks!